features/minion/core/features/src/main/resources/features.xml

```diff
@@ -27,5 +30,50 @@
     <feature name="minion-shell" description="Minion :: Shell" version="${project.version}">
         <feature>minion-core</feature>
         <bundle>mvn:org.opennms.features.minion/shell/${project.version}</bundle>
+        <feature version="[4.0,4.1)">spring</feature>
```

Comment:
Good job on putting this list together, but there are way too many dependencies here. These should be broken out into separate features, many of which already exist.